src/index/components/AvatarMenu.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <div class="dropdown-container">
+    <button
+      class="avatar-menu"
+      @click="openDropdown()"
+    >
+      <div v-html="avatar" />
```

Comment:
is there a way to remove the `v-html` directive? it can lead to XSS attacks

Reply:
done